charts/rancher-external-dns/v0.2.0/Chart.yaml

```diff
@@ -12,3 +12,4 @@ engine: gotpl
 maintainers:
 - name: rabadin
   email: rabadin@cisco.com
+kubeVersion: '< 1.22.0-0'
```

Comment:
same comment as above, but it seems like based on https://github.com/rancher/rancher/issues/35909 it would have failed to install anyways. So this change LGTM